weave-js/src/components/CodeEditor.tsx

```diff
@@ -84,6 +86,7 @@ export const CodeEditor = ({
       handleMouseWheel: handleMouseWheel ?? false,
       alwaysConsumeMouseWheel: alwaysConsumeMouseWheel ?? true,
     },
+    wordWrap: wrapLines ? ('on' as const) : ('off' as const),
```

Comment:
why the `as const` here?

Reply:
Short answer is that `tsc` wasn't able to determine that `on | off` is a subset of `on | off | wordWrapColumn | bounded`